ql/src/test/results/clientpositive/llap/explainuser_2.q.out

```diff
@@ -650,266 +650,267 @@ POSTHOOK: Output: hdfs://### HDFS PATH ###
 Plan optimized by CBO.
 
 Vertex dependency in root stage
-Map 11 <- Union 12 (CONTAINS)
-Map 14 <- Union 12 (CONTAINS)
-Map 16 <- Union 17 (CONTAINS)
-Map 21 <- Union 17 (CONTAINS)
-Map 22 <- Union 19 (CONTAINS)
-Map 23 <- Union 24 (CONTAINS)
-Map 30 <- Union 24 (CONTAINS)
-Map 31 <- Union 26 (CONTAINS)
-Map 32 <- Union 28 (CONTAINS)
-Reducer 10 <- Reducer 20 (SIMPLE_EDGE), Reducer 9 (SIMPLE_EDGE), Union 4 (CONTAINS)
-Reducer 13 <- Union 12 (SIMPLE_EDGE)
-Reducer 18 <- Union 17 (SIMPLE_EDGE), Union 19 (CONTAINS)
-Reducer 2 <- Map 1 (SIMPLE_EDGE), Map 15 (SIMPLE_EDGE)
-Reducer 20 <- Union 19 (SIMPLE_EDGE)
-Reducer 25 <- Union 24 (SIMPLE_EDGE), Union 26 (CONTAINS)
-Reducer 27 <- Union 26 (SIMPLE_EDGE), Union 28 (CONTAINS)
-Reducer 29 <- Union 28 (SIMPLE_EDGE)
-Reducer 3 <- Reducer 13 (SIMPLE_EDGE), Reducer 2 (SIMPLE_EDGE), Union 4 (CONTAINS)
-Reducer 5 <- Union 4 (SIMPLE_EDGE), Union 6 (CONTAINS)
-Reducer 7 <- Union 6 (SIMPLE_EDGE)
-Reducer 8 <- Reducer 2 (SIMPLE_EDGE), Reducer 29 (SIMPLE_EDGE), Union 6 (CONTAINS)
-Reducer 9 <- Map 1 (SIMPLE_EDGE), Map 15 (SIMPLE_EDGE)
+Map 10 <- Union 11 (CONTAINS)
```

Comment:
should we keep `hive.optimize.merge.adjacent.union.distinct` = false for the existing tests? You've added a new test anyways

Reply:
Okay. I add `set hive.optimize.merge.adjacent.union.distinct=false` in explainuser_2.q and unionDistinct_3.q to preserve their output files.

Reply:
👍